packages/debian/postinst

```diff
@@ -239,22 +242,22 @@ register_reference_extensions() {
 
 register_adu_user_and_process_with_eis() {
         if getent group "aziotid" > /dev/null; then
-            usermod -aG aziotid $adu_user
+            usermod -aG aziotid $ais_user
```

Comment:
@nihemstr, On Ubuntu Core, we can't create arbitrary user and group. The special user and group (snap_aziot_du) has been white-listed and created specifically for DU Agent snap. I discussed this off-line with @shiyi-peng and we will revert this change since postinst script is intended to be used during Dabien package installation only.

For DU Snap, the principal file will be created during Snap installation using 'configure  hook'